### ISSUE TITLE
Do not log sensitive data

### ISSUE THREAD

**Issue by Strubbl** [jangernert/FeedReader#845]
Please do not print sensitive data like e.g. passwords to the debug output. 

just saw this in the console:

```
(feedreader:27674): feedreader-DEBUG: 00:30:57.961: {"op":"login","user":"bernd","password":"555nose"}
```

**Comment by Strubbl** [comment]
Of course. Sorry, i forgot to mention that. It is the tt-rss plugin which i have tried.

**Comment by brendanlong** [comment]
Fixed in v2.8.0:

```
(feedreader:31236): feedreader-CRITICAL **: 14:20:18.619: Error response from TT-RSS API
URL: https://localhost/api/
Request object: {"op":"login","user":"test","password":"[redacted]"}
Response: 
```
